manifests/charts/kube-bench/values.yaml

```diff
@@ -38,8 +38,13 @@ pod:
     "helm.sh/hook-delete-policy": before-hook-creation
 
 image:
+<<<<<<< Updated upstream
```

Comment:
Looks like a bad merge

Reply:
Not sure why it appeared like this. I copied and pasted the line. Maybe it didn't like the format. I'll fix this.